src/main/resources/assets/minebigpony/lang/en_us.lang

```diff
@@ -0,0 +1,19 @@
+minebp.options.title=Big Pony Settings
```

Comment:
I would just call this `bigpony.xxx.xxx`

Reply:
Nevermind, leave it. I started it at the first commit.

Reply:
*shrug* I decided to keep it consistent with minelp.

Reply:
Mine Little Pony and Mine Big Pony. Thought that was what you were going for.